WIP: code cleanup

natebird opened the pull request:
- Fixed inconsistent indentation
- Fixed Travis-CI configuration to use a distro with postgresql installed
- Removed extra white spacing

natebird committed:
Change to vapor url. It was redirecting anyway.

natebird committed:
Fix whitespace issues.

natebird committed:
fix test failure text on parameterization test.

natebird committed:
fix indentation.

natebird committed:
combine cases to reduce duplication.

natebird committed:
cleanup more whitespace.

natebird committed:
Update travis config to use a language distro that comes with postgresql preinstalled.

tanner0101 commented on Package.swift:
```diff
@@ -3,7 +3,7 @@ import PackageDescription
     name: "PostgreSQL",
     dependencies: [
-   		 .Package(url: "https://github.com/qutheory/cpostgresql.git", majorVersion: 0),
+   		 .Package(url: "https://github.com/vapor/cpostgresql.git", majorVersion: 0),
```
We need to give this a minor.

natebird replied:
Yep. Updated below.

natebird committed:
Add minor version number to `cpostgresql` package.

natebird committed:
See if this installs postgresql on macOS.

natebird committed:
It says it is already installed. Maybe try starting it?

natebird committed:
Updated config as recommended by Travis support

This is the recommended configuration from Travis support to work on both Linux and OS X.

natebird commented:
Hmm. I might need to adjust the order a little bit. They had the export above the psql command.

natebird committed:
Put export before psql command

natebird commented:
Grrr. Will contact Travis support again

tanner0101 commented:
Let's skip travis for now. Great work on this!
